src/adapter/src/coord/dataflows.rs

```diff
@@ -638,7 +638,7 @@ fn eval_unmaterializable_func(
         }
         UnmaterializableFunc::CurrentTimestamp => pack(Datum::from(session.pcx().wall_time)),
         UnmaterializableFunc::CurrentUser => pack(Datum::from(session.user())),
-        UnmaterializableFunc::MzEnvironmentId => pack(Datum::from(state.config().environment_id)),
+        UnmaterializableFunc::MzEnvironmentId => pack(Datum::from(&*state.config().environment_id)),
```

Comment:
This is a type change. I assume we don't care yet!

Reply:
We do not! This seems good and fine! I think there is a `ScalarType::Uuid` somewhere that describes the output type of this function that needs to be updated to `ScalarType::String`, though.